src/components/Card/index.tsx

```diff
@@ -4,55 +4,52 @@ import styled from 'styled-components';
 import { DefaultProps, RenditionSystemProps } from '../../common-types';
 
 import asRendition from '../../asRendition';
-import { px } from '../../utils';
+import { DismissableContainer } from '../../internal/DismissableContainer';
 import { Box } from '../Box';
 import Divider from '../Divider';
 import { Flex } from '../Flex';
 import Txt from '../Txt';
 
-const Wrapper = styled.div<{ minHeight?: string }>`
-	border: solid 1px #dfdede;
-	border-radius: ${props => px(props.theme.radius || 3)};
-	box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.14);
-	padding: 20px;
-
-	background-color: #ffffff;
-	min-height: ${props => props.minHeight || 'auto'};
+const Wrapper = styled(DismissableContainer)<{ minHeight?: string }>`
+	& {
+		min-height: ${props => props.minHeight || 'auto'};
+		padding: ${props => (props.small ? '15px 18px' : '30px 36px')};
```

Comment:
We can probably do 3 -> 16px for both vertical and horizontal, and 4 -> `38px` for the large one so that we don't have a special case here.

Reply:
I picked up the "large" dimensions from the website, and just halved them since they don't make sense in smaller screens. Let's see what @apexider thinks

Reply:
@dimitrisnl where can i review that?